REQUIRE

```diff
@@ -1,2 +1,3 @@
 julia 0.6
 MathProgBase 0.7 0.8
```

Comment:
I should update exact cut prunings to MOI :p

Reply:
Yes :)
I will tag a last version of StructDualDynProg first with MPB that supports Julia v1.0 though since JuMP v0.19 is not released yet :-P